backend/main/src/Shared/Utils/Errors.ts

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * This is base class of use case.
```

Comment:
Is this comment supposed to be "base class of error"? It's up to you if you modify this comment because it doesn't show in the TypeDoc.

Reply:
This is an old comment. Sorry, I forgot to modify it. The correct comment is the following.
This is the base class error of this application.